src/tests/functional/plugin/conformance/test_runner/api_conformance_runner/src/ov_plugin/auto_batching_tests.cpp

```diff
@@ -1,25 +1,26 @@
 // Copyright (C) 2018-2023 Intel Corporation
 // SPDX-License-Identifier: Apache-2.0
 //
-#include <behavior/plugin/auto_batching_tests.hpp>
+#include "behavior/ov_plugin/auto_batching_tests.hpp"
```

Comment:
Why have you moved these tests to api conformance?? We validate `Auto` over template without any specific. Looks like these tests related to auto only. Conformance can be run over any device ("CPU", "AUTO:TEMPLATE" and etc). We do not hardcode any device specific in conformance